specification/columns/relatedresourceid.md

```diff
@@ -0,0 +1,31 @@
+# Related Resource ID
+
+Related Resource ID is populated when a usage resource line item is impacted by another line item that modifies its billing behaviour, e.g. when the charge has consumed a [*capacity reservation*](#glossary:capacity-reservation).
+
+The RelatedResourceID column MUST be present in the FOCUS data when the provider supports *capacity-reservations*. This column MUST be of type String, MUST be null when the charge is not related to another line item that impacts its billing behaviour, and MUST NOT be null when the charge is related to another line item that impacts its billing behaviour e.g a capacity reservation. The RelatedResourceID MUST match the ResourceID of thhe line item that has impacted the billing behaviour.
```

Comment:
We need to allow that some providers will support Capacity reservations, but will not support labeling of the usage line items with the ARN of a capacity reservation (which is all providers currently).

It's for this reason that I dont think this column is strictly necessary yet.